test/check-terminal

```diff
@@ -26,6 +26,18 @@ class TestTerminal(MachineCase):
         m = self.machine
         b.default_user = "admin"
 
+        # XXX - figure out why locale isn't set up properly by default
+        if "debian" in m.image:
+            m.execute("echo 'en_US.UTF-8' >>/etc/locale.gen; locale-gen")
+            m.execute("echo 'export LANG=en_US.UTF-8' >>/home/admin/.bashrc")
```

Comment:
Without this, non-ASCII characters are not handled as expected.  This is not limited to our terminal, all terminals are 'broken' in the same way.

I am pretty sure this is an artifact of the debian-8 virt-builder templates.  A normal Debian install should have properly configured locales.